examples/event-counters/Examples.EventCounters/Program.cs

```diff
@@ -0,0 +1,41 @@
+// <copyright file="Program.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using System.Diagnostics.Tracing;
+using OpenTelemetry;
+using OpenTelemetry.Metrics;
+
+// Create EventSources and EventCounters
+using EventSource eventSource = new("MyEventSource");
+using EventCounter eventCounter = new("MyEventCounter", eventSource);
+using PollingCounter pollingCounter = new("MyPollingCounter", eventSource, () => new Random().NextDouble());
```

Comment:
Create an instance of `Random` class and use that in the `Func<double>` instead of creating a new instance of `Random` class on every poll.

It would still work but we are incorporating good practices in our examples anyway.